streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -401,17 +355,11 @@ public void shouldOnlyNeedCommitWhenChangelogOffsetChanged() {
 
         task.prepareCommit();
         task.postCommit(true);
-
-        EasyMock.verify(stateManager);
     }
 
     @Test
     public void shouldThrowOnCloseCleanError() {
-        EasyMock.expect(stateManager.changelogOffsets()).andStubReturn(Collections.emptyMap());
-        stateManager.close();
-        EasyMock.expectLastCall().andThrow(new RuntimeException("KABOOM!")).anyTimes();
-        EasyMock.expect(stateManager.changelogPartitions()).andReturn(Collections.singleton(partition)).anyTimes();
```

Comment:
A majority of these setup statements have also been removed because according to Mockito they do not get called. I have further confirmed that removing these statements in the EasyMock world does not break the tests.